client-js/app/elements/labrad-nodes.ts

```diff
@@ -63,8 +63,15 @@ export class LabradInstanceController extends polymer.Base {
     return places.serverUrl(instanceName);
   }
 
-  ready() {
-    this.updateButtonState(this.status);
+  attached() {
+    this.async(() => {
+      this.fire('labrad-instance-controller::ready', {
```

Comment:
Does the double colon here have any special meaning? Or are the names here arbitrary strings and this is just to namespace things?